cypress/integration/plugins/observability-dashboards/7_app_analytics.spec.js

```diff
@@ -453,6 +254,7 @@ describe('Application Analytics home page', () => {
       .within(() => {
         cy.get('.euiCheckbox').click();
       });
+    cy.wait(delayTime);
```

Comment:
Isn't there a better way to implement this? `wait(ms)` should be almost universally avoided. The sony exception i can think of is when debouncing is involved.